config/application.yml.default

```diff
@@ -285,6 +285,7 @@ set_remember_device_session_expiration: false
 sp_handoff_bounce_max_seconds: 2
 show_user_attribute_deprecation_warnings: false
 show_account_recovery_recovery_options: false
+system_demand_report_configs: '{"email": "foo@bar.com"}'
```

Comment:
are we only going to have a single email here?
I'd expect a list of emails here, since it's rare we only ever want to send a report to a single person

Reply:
1. if the config is only a single email, maybe just make the config a simple string instead of JSON object, less error-prone
2. I don't think we should check that email into the repo, or post it here, because of spam concerns